context/app/static/js/components/Header.jsx

```diff
@@ -34,12 +45,34 @@ export default function Header() {
           <a href="/">
             <HubmapLogo className={classes.hubmaptypeLight} aria-label="HubMAP logo" />
           </a>
-          <Typography variant="h5" className={classes.title} />
           {['Donor', 'Sample', 'Dataset'].map((type) => (
             <Button key={type}>
               <a href={`/search?entity_type[0]=${type}`} className="navLink">{`${type}s`}</a>
             </Button>
           ))}
+
+          <Typography variant="h5" className={classes.title} />
+
+          <Button ref={anchorRef} onClick={toggle} style={{ color: 'white' }}>
+            Showcases
+            {open ? <ArrowDropDownIcon /> : <ArrowDropUpIcon />}
+          </Button>
+          <Popper open={open} anchorEl={anchorRef.current} placement="bottom-start">
+            <Paper>
+              <ClickAwayListener onClickAway={toggle}>
+                <MenuList id="showcase-options">
+                  {['Spraggins'].map((showcaseName) => (
```

Comment:
Since this is in the `Header`, and it takes no props, I don't think there is a particularly easy way to do this without changing a bunch of other code.  Every page would need to need to send this information if we continue to use server-side rendering for markdown. I think the main reason I decided to go with server-side is because of the front-matter and storing it in a sensible way.  There is [no great way](https://github.com/mdx-js/mdx/issues/199) to do this sort of thing in `mdx` short of creating a bunch of `const`s.  I liked frontmatter, so I went this route.

Reply:
We could store it in a `constants` file maybe?

Reply:
Lets just leave it as is, and if it becomes a pain to update, we'll find something better. Sorry for the bother!